projects/developer/src/app/data/metrics/component.scss

```diff
@@ -1,3 +1,11 @@
+::ng-deep .metricsDateFilter {
+    z-index: 10021004 !important;

```

Comment:
oddly specific?

Reply:
PrimeNG sets the z-index for the overlay sidebar panel thing to 10021001, for whatever reason. Calendar date picker had to go in front of this.